src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -167,7 +167,7 @@ public function hasVariableOfName($nodes, string $name): bool
     /**
      * @param Node|Node[] $nodes
      */
-    public function findClassMethodOfName($nodes, string $name): ?ClassMethod
+    private function findClassMethodOfName($nodes, string $name): ?ClassMethod
```

Comment:
This method is not used outside of class but I'd like to keep it public for further usage. @TomasVotruba, is there any way to skip it?

Reply:
Why make it `public` if it's not used?

Reply:
After some changes, it looks like we don't need both of the added methods that is why I removed them. Nevertheless, I consider such classes as some "helper" libs that have some API. Maybe some methods are not used at the moment even inside the class but they can be helpful for someone else.

I know your answer and more or less agree with you :D